ArcGISRuntimeSDKQt_CppSamples/Maps/ReadGeopackage/ReadGeopackage.h

```diff
@@ -0,0 +1,74 @@
+// [WriteFile Name=ReadGeoPackage, Category=Maps]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifndef READGEOPACKAGE_H
+#define READGEOPACKAGE_H
+
+namespace Esri
+{
+namespace ArcGISRuntime
+{
+class Map;
+class MapQuickView;
+class listModel;
+class listView;
+class GeoPackage;
+class Layer;
+class RasterLayer;
+class FeatureTable;
+class FeatureLayer;
+class GeoPackageRaster;
+class GeoPackageFeatureTable;
+
```

Comment:
Remove empty line

Reply:
And there would be 2-space indenting for the nested namespace and declared classes.